lightning/src/offers/offer.rs

```diff
@@ -376,8 +375,7 @@ pub struct Offer {
 ///
 /// [`InvoiceRequest`]: crate::offers::invoice_request::InvoiceRequest
 /// [`Bolt12Invoice`]: crate::offers::invoice::Bolt12Invoice
-#[derive(Clone, Debug)]
-#[cfg_attr(test, derive(PartialEq))]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
The sub-structs can be kept as they were since `Offer` and `Refund` have custom `PartialEq` implementations.

Reply:
Discussed offline. We should be able to access any fields needed from the offer without exposing the contents.